content/builds/rifle-mech.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Mechknight [pDPS]"
+description = "4 button rotation with lots of Signets"
+date = 2024-08-27
+draft = false
+template = "build.html"
+
+[taxonomies]
+categories = ["group"]
+tags = ["power","engineer","mechanist","eod","lowcog","lowphys","lowrep"]
+authors = ["scooby"]
+specs = ["mechanist", "engineer"]
+
+[extra]
+series = "engineer"
+tagline = "Reports of my death have been greatly exaggerated"
+keywords = "Guild Wars 2, GW2, LI, Mechanist"
+toc = true
+balance = "August 2024"
+benchmark = "~28k DPS"
+hidden = "true"
++++
+
+## Notes
+
+---
+
+Extremely simple build that does almost all of it's damage from range.
+
+Highly recommended, beginner friendly build that can be played in all scenarios, whether it be instanced content such as fractals, raids, or strikes, or if you just need a safe, comfortable build for open world.
+
+## Gearing
+
+{{ medium(stat="Berserker's", rune="Scholar") }}
+{{ rifle(stat="Berserker's", sigils=["Force", "Air"]) }}
+{{ trinkets(stat="Berserker's", relic="Thief") }}
+
+---
+
+- Consumables:
+  - Bowl of Sweet and Spicy Butternut Squash Soup OR Cilantro Lime Sous-Vide Steak (Power/Ferocity)
```

Comment:
I haven't done an easy macro to fit it into a table yet, but we can do `{{ item(id="12345") }}` for consumables if you want.